test/e2e/windows/density.go

```diff
@@ -30,21 +30,14 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/kubernetes/test/e2e/framework"
-	imageutils "k8s.io/kubernetes/test/utils/image"
```

Comment:
All image names should be imported from here for consistency

Reply:
👍